README.md

```diff
@@ -30,6 +29,12 @@ It is used in [Yii Framework] but can be used separately.
 [![static analysis](https://github.com/yiisoft/db/actions/workflows/static.yml/badge.svg?branch=dev)](https://github.com/yiisoft/db/actions/workflows/static.yml)
 [![type-coverage](https://shepherd.dev/github/yiisoft/db/coverage.svg)](https://shepherd.dev/github/yiisoft/db)
 
+## Usage 
+
+For use read the [docs](/docs/en/getting-started.md).
```

Comment:
```suggestion
[Check the documentation docs](/docs/en/getting-started.md) to learn about usage.
```

I'd leave some basics here with really cool syntax or something to have a good overview for first-time user.